src/util/Util.js

```diff
@@ -84,6 +84,11 @@ define(['okta'], function (Okta) {
       .uniq()
       .value();
   };
+  
+  //helper to call setTimeout
+  Util.callAfterTimeout = function (callback, time) {
```

Comment:
I'm using it to help mock the setTimeout in my jasmine test. In the signin widget i see Q used more often than setTimeout, but we do have setTimeout in other places. However, we never used setTimeout with this large of a delay, which made it difficult to test.